api/handlers_metrics.go

```diff
@@ -3,22 +3,30 @@ package api
 import (
 	"fmt"
 	"net/http"
-	"sort"
 	"strings"
 	"time"
 
 	"github.com/YaleSpinup/cost-api/apierror"
 	"github.com/YaleSpinup/cost-api/cloudwatch"
 	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 )
 
-// MetricsGetImageHandler gets metrics from cloudwatch
-func (s *server) MetricsGetImageHandler(w http.ResponseWriter, r *http.Request) {
+// GetEC2MetricsImageHandler gets metrics from cloudwatch
+func (s *server) GetEC2MetricsImageHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Why even have these Image handlers? Isn't it better to just use the URL ones which have caching?

Reply:
Good point, I guess we don't really use it.